source/macos/AdaptiveCards/AdaptiveCards/Views/ACRTextView.swift

```diff
@@ -87,6 +87,22 @@ class ACRTextView: NSTextView, SelectActionHandlingProtocol {
         }
     }
     
+    func customizeForFactSetCell(isTitleView: Bool, config: RenderConfig) {
+        setContentCompressionResistancePriority(.required, for: .vertical)
+        alignment = .left
+        isEditable = false
+        backgroundColor = .clear
+        if isTitleView {
+            let fontManager = NSFontManager.shared
+            font = fontManager.convert(font ?? NSFont.systemFont(ofSize: config.inputFieldConfig.font.pointSize), toHaveTrait: NSFontTraitMask.boldFontMask)
```

Comment:
```suggestion
            let fontManager = NSFontManager.shared
            font = fontManager.convert(font ?? NSFont.systemFont(ofSize: 14), toHaveTrait: NSFontTraitMask.boldFontMask)
```
I dont think we should use `InputFieldConfig` values here. Maybe a hardcoded value if `font` is nil should be fine in this case.